src/robusta/cli/main.py

```diff
@@ -65,16 +54,13 @@ class GlobalConfig(BaseModel):
 
 class HelmValues(BaseModel, extra=Extra.allow):
     globalConfig: GlobalConfig
-    sinksConfig: List[
-        Union[
-            SlackSinkConfigWrapper, RobustaSinkConfigWrapper, MsTeamsSinkConfigWrapper
-        ]
-    ]
+    sinksConfig: List[Union[SlackSinkConfigWrapper, RobustaSinkConfigWrapper, MsTeamsSinkConfigWrapper]]
     clusterName: Optional[str]
     isSmallCluster: Optional[bool]
     enablePrometheusStack: bool = False
     disableCloudRouting: bool = False
     enablePlatformPlaybooks: bool = False
+    # TODO: Discuss this None default value
```

Comment:
reminder

Reply:
It relates to a annotations and was solved in a different PR, will just remove it